### PR TITLE
app-crypt/sbctl: Back-port bashism in kernel install hook

### PR DESCRIPTION
**Pull request by imwints**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-crypt/sbctl/files/sbctl-0.14-installkernel-bashism.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
+diff --git a/contrib/kernel-install/91-sbctl.install b/contrib/kernel-install/91-sbctl.install
```
Comment (ajakk) [inline app-crypt/sbctl/files/sbctl-0.14-installkernel-bashism.patch:1]: Thanks! Can you add upstream/downstream references according to https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#how?

**Review by ajakk — CHANGES_REQUESTED** [review]

File: app-crypt/sbctl/files/sbctl-0.14-installkernel-bashism.patch (hunk 1 above)
Comment (imwints) [reply app-crypt/sbctl/files/sbctl-0.14-installkernel-bashism.patch]: I hope this is the expected form for the patch

**Comment by imwints** [comment]
Uh I messed something up when I first created the PR. I somehow created the first branch off of a borked master or something like that. I'm not a git wizard but I think I fixed it. However if you look at Githubs force-push-compare thingy there are like 200k lines changed. Is this right?

**Comment by ajakk** [comment]
Thanks! Fixed up the `Closes: Closes:` on merge.

**Pull request closed without merging**